Implemented "showFromRect:inView:animated:"

michaeltyson opened the pull request:
Hi! This adds the ability to select an origin rect, and have JGActionSheet automatically determine which side to display on. This mirrors the functionality from UIActionSheet.

michaeltyson committed:
Implemented "showFromRect:inView:animated:"

This adds the ability to select an origin rect, and have JGActionSheet automatically determine which side to display on. This mirrors the functionality from UIActionSheet.

JonasGessner commented:
Awesome, this is something I wanted to implement a long time ago. I'll test it when I get some free time but it looks promising! Wicked!

michaeltyson commented:
Cool =)

michaeltyson committed:
Implemented "moveToRect:animated:"

This provides the equivalent behaviour to moveToPoint..., but automatically selects the arrow direction and anchor point.

michaeltyson commented:
Just added an additional commit that provides the "moveToRect" functionality necessary when rotating

tposch commented:
I would love to get this pull-requested merged and the pod updated. Thanks @michaeltyson for implementing it.

JonasGessner commented:
Thanks for reminding me. I'm super busy but I hope that I will be able to test this soon.

michaeltyson commented:
Ah, the downside of getting pull requests - yet another thing to add to the todo list =) I've always got a backlog on my projects.
